apps/src/turtle/blocks.js

```diff
@@ -697,8 +701,8 @@ exports.install = function(blockly, blockInstallOptions) {
   };
 
   blockly.Blocks.draw_move.DIRECTIONS = [
-    [msg.moveForward(), 'moveForward'],
-    [msg.moveBackward(), 'moveBackward']
+    [msg.forward(), 'moveForward'],
+    [msg.backward(), 'moveBackward']
```

Comment:
It wasn't clear to me why this change was needed, so I spent a bit of time tracking down why the old options were `"move forward by"` and `"move backward by"` but they show up like this:

![image](https://user-images.githubusercontent.com/413693/64446309-549c3400-d08d-11e9-8d19-0ce9d87ee0d9.png)

It turns out there's [already code](https://github.com/code-dot-org/blockly/blob/7c93fa1faf602f9c372294f335e04d9ebaadf8cf/core/ui/fields/field_dropdown.js#L193) to make the dropdown only show the _difference_ between two strings. As an example:

![image](https://user-images.githubusercontent.com/413693/64446374-7990a700-d08d-11e9-8f48-f969b4fdc4b2.png)

Looks like it takes the commonality out of both strings and displays it as only text, so the dropdown only includes the difference:

![image](https://user-images.githubusercontent.com/413693/64446447-af359000-d08d-11e9-8479-a32e72f67ed5.png)

Does this affect the plan? When should we prefer interpolation vs. extracting common words out of the dropdown?

Reply:
So it's a difference between having the strings

```json
{
  "forward": "forward",
  "backward": "backward",
  "moveBy": "move %1 by %2 pixels"
}
```

vs

```json
{
  "moveForwardBy": "move forward by %1 pixels",
  "moveBackwardBy": "move backward by %1 pixels"
}
```

?

Reply:
If you use `""move forward by %1 pixels"` and `""move backward by %1 pixels"` I _think_ the block will display "by %1 pixels" as label text. Not sure if there's a way to extract common words, then interpolate those words as inputs.

Reply:
hmm, in that case I don't see how we could leverage the extracting common words functionality here

Reply:
Right, I don't think we necessarily need to. Interpolation seems better, because someone can't accidentally translate the prefix/suffix in different ways. Mostly want to [understand why it changed](https://github.com/code-dot-org/blockly-core/commit/b6f32189a989cbf0f9dee4abfca0a9d109ec6b62), and what our stance is going forward.

Reply:
(And if we go the interpolation route, let's clean up the old dropdown strings.)

Reply:
I made a few edits to the code and I haven't been able to extract the common words, interpolation will be the option to go with.